docs/scale.md

```diff
@@ -272,125 +339,14 @@ Kubernetes cluster, based on the number of users in your organization:
 ## Using heterogeneous node pools with `nodeSelector`
```

Comment:
isn't this already documented in configure.md? (Assign resource-hungry pods to larger nodes)

Can we unify that doc?